### ISSUE TITLE
Why can't CodeCrunch run my code when I package the Main Classes?

### ISSUE THREAD

**Issue by RachelCheah** [nus-cs2030/2021-s2#267]
# Summary
Sorry I'm not too familiar with how packages work, but why can't I package everything into the same package (including the main classes)? CodeCrunch just returns me an empty output as shown below.

## Screenshots: 
![image](https://user-images.githubusercontent.com/55133167/115350968-5118da80-a1e8-11eb-8315-82088d29d22a.png)

**Comment by yx0555** [comment]
I think code crunch requires you to put all the files(Event classes etc.) inside the package while your main classes are outside. Remember to import the files over to your main classes by doing "import cs2030.simulator.(class name)" at the top!

**Comment by gwajoon** [comment]
Hi @RachelCheah , I had the same issue as well. As what @yx0555 mentioned, you have to move all the main classes out of your cs2030.simulator package. Then, import whatever classes you need to run your main classes accordingly and upload them all to CodeCrunch! That should fix the problem :)

**Comment by RachelCheah** [comment]
Not sure if anyone needs this but I found out that I can do this instead of needing to rewrite the Main methods to import the classes from the package. I simply created another class that is outside of the package to call the Main class inside the package and CodeCrunch accepts this.

For example, I renamed my initial Main1 class to Main1Simulator and created another Main1.java to import the initial Main as follows :

```
import cs2030.simulator.Main1Simulator;

public class Main1 {
    public static void main(String[] args) {
        cs2030.simulator.Main1Simulator.main(null);
    }
}
```

Thereby, allowing CodeCrunch to run this properly and give an output. Hope this helps someone so that you don't have to rewrite your code :)
